modules/accessing-the-administrator-perspective.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// web_console/about-administrator-perspective.adoc
+
+:_content-type: PROCEDURE
+[id="accessing-the-administrator-perspective_{context}"]
+= Accessing the Administrator perspective
+
+.Prerequisites
+To access the Administrator perspective, ensure that you have logged in to the web console and have the administrator role privileges.
+
+The *Administrator* perspective in the {product-title} web console provides workflows specific to administrator use cases.
+
+You can access the *Administrator* perspective from the web console as follows:
```

Comment:
I would reccomend moving this around to follow the mod doc guidelines: https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs

```suggestion
The *Administrator* perspective in the {product-title} web console provides workflows specific to administrator use cases. You can access the *Administrator* perspective from the web console as follows:

.Prerequisites
To access the Administrator perspective, ensure that you have logged in to the web console and have the administrator role privileges.
```